x-pack/plugins/transform/public/app/common/request.ts

```diff
@@ -47,9 +48,15 @@ export interface SimpleQuery {
   };
 }
 
-export type PivotQuery = SimpleQuery | SavedSearchQuery;
+export interface FilterBasedSimpleQuery {
+  bool: {
```

Comment:
Is this a change from the previous 'empty' query that was applied `"match_all": {}` ? This is causing a side effect when cloning a transform, in that the search field shows the 'advanced' JSON version

<img width="449" alt="image" src="https://user-images.githubusercontent.com/7405507/215845177-6be22742-a90b-4e6e-8441-2fc891b574a4.png">


rather than the preferable empty KQL bar.

Reply:
Added a note for a follow up in #146187.

Reply:
PR is up to fix this issue here: #151665